features/in-app-install.yml

```diff
@@ -0,0 +1,12 @@
+name: In-app install experience
```

Comment:
I'm not very excited about this name. It's not very specific and it doesn't seem to be widely used (or perhaps used at all) in the existing blog posts and docs about this. I think good alternatives might be (in my order of preference):

- beforeinstallprompt
- App install prompt
- App install prompt event

In any case, the ID would need to change too.